lua/scratch/config.lua

```diff
@@ -29,7 +29,7 @@ end
 
 -- TODO: #21 allow Lua configuration at the same time
 local default_config = {
-  scratch_file_dir = vim.fn.stdpath("cache") .. slash .. "scratch.nvim",
+  scratch_file_dir = vim.fn.stdpath("cache"):gsub(vim.env.HOME, '~') .. slash .. "scratch.nvim",
```

Comment:
we still using gsub?

Reply:
@LintaoAmons This line is used for rendering the `scratch_config.scratch_file_dir` to have a `~` rather than the full, absolute path of the `scratch_file_dir`. `vim.fs.normalize` only work by converting the `~` to the home directory and not the other way around, so it doesn't make sense to use `normalize` here.

Is there a different method that works better for this case?

Reply:
heh I also don't have Windows...

I think this is a fair point. I can remove this and simply reduce the entire pull request to the change in `M.getConfig` to normalize the `scratch_file_dir`.